soccer/radio/USBRadio.cpp

```diff
@@ -16,7 +16,7 @@ using namespace std;
 using namespace Packet;
 
 // Timeout for control transfers, in milliseconds
-static const int Control_Timeout = 1000;
+static const int Control_Timeout = 10;
```

Comment:
This seems like an extremem change. This fixed radio probs?

Reply:
Not completely, but when it does happen now, it's much less disruptive

Reply:
1s is a bit too high imo anyway, I don't think lowering the timeout to this is a bad idea, unless we expect transfers to take >10ms. If >10ms is abnormal, I don't think it hurts at all.